src/views/ContactMeView.vue

```diff
@@ -0,0 +1,159 @@
+            />
+            <n-gradient-text
+              :gradient="{
+                deg: 180,
+                from: 'rgb(85, 85, 85)',
+                to: 'rgb(170, 170, 170)',
+              }"
+            >
+              Spbgzh </n-gradient-text
+            ><br />
+            <n-tag :bordered="false" type="info" round>
+              Followers:
+              <n-number-animation
+                ref="numberAnimationInstRef"
+                :from="0"
+                :to="followers"
+              />
+              · Following:
+              <n-number-animation
+                ref="numberAnimationInstRef"
+                :from="0"
+                :to="following"
+              />
+              <template #icon>
+                <n-icon :component="PeopleOutline" />
+              </template>
+            </n-tag>
+            <a href="https://github.com/spbgzh">https://github.com/spbgzh</a>
+          </n-card>
+        </n-tab-pane>
+        <n-tab-pane name="Email" tab="Email">
+          <n-card title="Email" hoverable>
+            <template #cover>
+              <n-icon :size="350" :component="MailOutline" />
+            </template>
+            <n-gradient-text
+              :gradient="{
+                deg: 180,
+                from: 'rgb(85, 85, 85)',
+                to: 'rgb(170, 170, 170)',
+              }"
+            >
+              zjjhgzh@gmail.com
+            </n-gradient-text>
+          </n-card>
+        </n-tab-pane>
+        <n-tab-pane name="Wechat" tab="Wechat">
+          <n-card title="Wechat" hoverable>
+            <template #cover>
+              <n-icon :size="350" :component="LogoWechat" />
+            </template>
+
+            <n-gradient-text
+              :gradient="{
+                deg: 180,
+                from: 'rgb(85, 85, 85)',
+                to: 'rgb(170, 170, 170)',
+              }"
+            >
+              Spbgzh
+            </n-gradient-text>
+          </n-card>
+        </n-tab-pane>
+        <n-tab-pane name="VK" tab="VK">
+          <n-card title="VK" hoverable>
+            <template #cover>
+              <n-icon :size="350" :component="LogoVk" />
+            </template>
+
+            <n-gradient-text
+              :gradient="{
+                deg: 180,
+                from: 'rgb(85, 85, 85)',
+                to: 'rgb(170, 170, 170)',
+              }"
+            >
+              zjjhgzh
+            </n-gradient-text>
+          </n-card>
+        </n-tab-pane>
+      </n-tabs>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import type { AxiosResponse } from "axios";
+import {
+  LogoGithub,
+  PeopleOutline,
+  MailOutline,
+  LogoWechat,
+  LogoVk,
+} from "@vicons/ionicons5";
+
+export default defineComponent({
+  data() {
+    return {
+      following: 0,
+      followers: 0,
+      public_repos: 0,
+    };
+  },
+  mounted() {
+    axios
+      .get("https://api.github.com/users/spbgzh")
+      .then((response: AxiosResponse) => {
```

Comment:
```suggestion
      .then((response) => {
```
 тайпскрипт должен автоматом вывести этот тип

Reply:
Этот тип используется как generic когда вы хотите явно описать что лежит внутр `response.data`